### PR TITLE
Bottom Menu

### PR DESCRIPTION
**Pull request by salmanmaredia**

### CODE REVIEW

**Review by louisbirla — CHANGES_REQUESTED** [review]
Looks good but definitely does not have all the components to complete the ticket

File: src/components/bottomMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                            menu.star_button.starred ? 'star' : 'star-outline')
+                    }
+                    {menu.notifications_enabled &&
+                        renderItem(
+                            menu.notifications_enabled ? 'Disable Notification' : 'Enable Notification',
+                            handleClose,
+                            null,
+                            menu.notifications_enabled ? 'bell' : 'bell-off')
+                    }
+                    {menu.delete &&
```
Comment (louisbirla) [inline src/components/bottomMenu.tsx:85]: Could you remove the delete button b/c there's no ticket & implementation is broken

**Review by louisbirla — CHANGES_REQUESTED** [review]

File: src/components/bottomMenu.tsx (hunk 1 above)
Comment (salmanmaredia) [reply src/components/bottomMenu.tsx]: Removed

**Review by louisbirla — APPROVED** [review]
Nice!

**Pull request merged**